doc/hld_template.md

```diff
@@ -0,0 +1,65 @@
+# HLD Name #
+
+## Table of Content 
+
+### 1. Revision  
+
+### 2. Scope  
+
+This section describes the scope of this high-level design document in SONiC.
+
+### 3. Definitions/Abbreviations 
+
+This section covers the abbreviation if any, used in this high-level design document and its definitions.
+
+### 1. Overview 
+
+The purpose of this section is to give an overview of high-level design document and its architecture implementation in SONiC. 
+
+### 2. Requirements
+
+This section list out the basic requirements for the HLD coverage and exemptions (not supported) if any for this design.
+
+### 3. Architecture Design 
+
+This section covers the changes that are required in the SONiC architecture and explains how the module/sub-module fits in the architecture. 
+
+### 4. High-Level Design 
+
+This section covers the high level design of the feature/enhancement. This section covers the following points in detail.
+		
+		- What are the modules and sub-modules that are modfiied for this design?
```

Comment:
Fix the typo - "modfiied" - generally I would suggest a spell check

Reply:
Done.

Reply:
Ack for this one - I still see one more spelling error below ("distruption")